tests/filters/test_filters.py

```diff
@@ -2,118 +2,29 @@
-                              np.array([[0, 0, 0, 0, 0],
-                                        [0, 1, 0.96, 0, 0],
-                                        [0, 0.67, 0.87, 0, 0],
-                                        [0, 0, 0, 0, 0],
-                                        [0, 0, 0, 0, 0]])),
-                            coo_matrix(
-                              np.array([[0, 1, 1, 1, 1],
-                                        [0, 1, 1, 1, 1],
-                                        [0, 1, 1, 1, 1],
-                                        [0, 1, 1, 1, 1],
-                                        [0, 0, 0, 0, 0]]))],
-                           3, [coo_matrix(
-                              np.array([[0, 0, 0, 0, 0],
-                                        [0, 1, 0, 0, 0],
-                                        [0, 0, 1, 0, 0],
-                                        [0, 0, 0, 0, 0],
-                                        [0, 0, 0, 0, 0]])),
-                               coo_matrix(
-                                   np.array([[0, 0, 0, 0, 0],
-                                             [0, 1, 0.96, 0, 0],
-                                             [0, 0.67, 0.87, 0, 0],
-                                             [0, 0, 0, 0, 0],
-                                             [0, 0, 0, 0, 0]]))]),
-                          ([coo_matrix(np.array([[1, 1, 1]]))], 3, []),
-                          ([coo_matrix(np.array([[0, 1, 1]]))], 3,
-                           [coo_matrix(np.array([[0, 1, 1]]))])])
-def test_filter_rois_by_longest_edge_length(coo_rois, longest_edge_thrsh,
-                                            expected_rois):
-    """
-    Test Cases (Included in the list of coo_matrices):
-    1. Larger than longest edge thrsh in x direction, contiguous
-    2. Larger than longest edge thrsh in y direction, contiguous
-    3. Larger than longest edge thrsh in x and y, contiguous
-    4. Larger than longest edge thrsh in x, non contiguous
-    5. Larger than longest edge thrsh in y, non contiguous
-    6. Larger than longest edge thrsh in x and y, non contiguous
-    7. Smaller than longest edge thrsh in x and y, contiguous
-    8. Smaller than longest edge thrsh in x and y, non contiguous
-    9. Empty list
-    10. List with more than one element two trues and one false
-    11. Singleton roi with ptp 2 and length equal to filter
-    12. Singleton roi with ptp 1 and length less than filter
-    """
-    filtered_rois = filter_longest_edge_length(coo_rois,
-                                               longest_edge_thrsh)
-    # assume that test is true and the indices line up between calculated and
-    # expected
-    for i in range(len(filtered_rois)):
-        assert np.allclose(filtered_rois[i].toarray(),
-                           expected_rois[i].toarray())
+def coos_by_aspect(threshold):
+    width = 100
+    height = int(width * threshold)
+    while (height/width) <= threshold:
```

Comment:
If width and threshold are set can't you just compute this?

Reply:
I was being lazy, I will change it

Reply:
done